WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteViewModel.kt

```diff
@@ -738,15 +745,29 @@ class MySiteViewModel @Inject constructor(
     }
 
     fun onPullToRefresh() {
-        postCardsSource.refresh()
+        refresh()
     }
 
+    @Suppress("unused", "UNUSED_PARAMETER")
```

Comment:
Minor (🔍): Consider removing the `unused` as it is actually used. However, the parameter it is indeed unused and as such the `UNUSED_PARAMETER` can stay.

Reply:
will do

Reply:
Removed in febc6f9